lightwood/mixers/nn.py

```diff
@@ -202,6 +219,9 @@ def _fit(self, train_ds, test_ds, stop_training_after_seconds=None):
         if self.optimizer_args is None:
             self.optimizer_args = {'lr': 0.0005}
 
+        if is_ts_target:
+            self.optimizer_args['weight_decay'] = 2e-2
```

Comment:
I've seen weight decay at 2e-5 to 1e-5; I trust it works but any reason its 2e-2?

Reply:
Purely empirical. I started _very_ aggresively and settled for this value, though I didn't test with smaller values, maybe I should?

Reply:
I'm ok with leaving it as 2e-2 but it may be worth allowing the user to specify it if it's not declared. I wonder if different data sets would do better with different LR but these are the magic numbers of optimization that as far as I know are all empirical anyway